# Action Constants
int8 ACTION_OPEN_GRIPPER=0
int8 ACTION_CLOSE_GRIPPER=1
int8 ACTION_MOVE_TO=2
int8 ACTION_MOVE_OVER=3
int8 ACTION_IDLE=4
int8 DONE_STACKING = 5
int8 action
int8 target
---
bool success
